project_event/security/project_security.xml

```diff
@@ -130,14 +119,26 @@
         <field name="groups" eval="[(4,ref('base.group_user'))]"/>
         <field name="active">False</field>
     </record>
-    <record id="calendar_event_access_project_editor_manager_all" model="ir.rule">
-        <field name="name">Calendar Event all access to editor and manager</field>
+
+    <record id="calendar_event_access_project_manager_all" model="ir.rule">
+        <field name="name">Calendar Event all access manager</field>
         <field name="model_id" ref="calendar.model_calendar_event"/>
         <field name="domain_force">[(1,'=',1)]</field>
-        <field name="groups" eval="[(4, ref('group_project_event_editor')),   (4, ref('group_project_event_manager'))]"/>
+        <field name="groups" eval="[(4, ref('group_project_event_manager'))]"/>
         <field eval="1" name="perm_unlink"/>
         <field eval="1" name="perm_write"/>
         <field eval="1" name="perm_read"/>
         <field eval="1" name="perm_create"/>
     </record>
+
+    <record id="calendar_event_access_project_editor_all" model="ir.rule">
+        <field name="name">Calendar Event all access to editor</field>
```

Comment:
not all access, only read access in this case

Reply:
1=1 i mean, access to all records

Reply:
For this kind of comment please just comment not request change so if you are not available we can still merge if someone else do the review